### PR TITLE
Fix test case so reduction is parellelized for validation.

### PR DESCRIPTION
**Pull request by csarofeen**

### CODE REVIEW

File: test/cpp/jit/test_gpu.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// #if defined(USE_CUDA)
+#if defined(USE_CUDA)
```
Comment (naoyam) [inline test/cpp/jit/test_gpu.cpp:1]: Is this change related to this fix?

**Review by naoyam — APPROVED** [review]

File: test/cpp/jit/test_gpu.cpp (hunk 1 above)
Comment (csarofeen) [reply test/cpp/jit/test_gpu.cpp:1]: no, think I commented it in the last PR and forgot to change it back.
